desktop/cmp/rest/impl/RestForm.scss

```diff
@@ -1,5 +1,7 @@
 .xh-rest-form {
-  &__control {
-    margin-bottom: var(--xh-pad-px);
+  width: 470px;
```

Comment:
Here we're using styles for an even more critical width - the overall width of the form - which I think is fine but again could argue for having the label width and other styles be done primarily in CSS as well.